lib/app/router/chat_routes.dart

```diff
@@ -118,3 +118,16 @@ class CreateGroupModalRoute extends BaseRouteData {
           type: IceRouteType.bottomSheet,
         );
 }
+
+@TypedGoRoute<GroupRoute>(
+  path: '/group/:pubkey',
+  routes: [],
```

Comment:
do we really need to define an empty list of the routes here?

Reply:
Nope, removed.